spec/presenters/two_factor_options_presenter_spec.rb

```diff
@@ -76,5 +76,16 @@
         ]
       end
     end
+    context 'when priority is authentication first in a/b testing bucket' do
+      let(:presenter) do
+        described_class.new(user_agent: user_agent, priority_bucket: :authentication_app_priority)
+      end
+
+      it 'supplies all authn as the first option' do
```

Comment:
Nit: Is this test description accurate?

```suggestion
      it 'supplies auth app as the first option' do
```